.github/workflows/main.yml

```diff
@@ -1,6 +1,6 @@
 name: Build
 
-on: 
```

Comment:
Formatting fixes should be a separate commit, this touches way too much for a single commit.